app/page/index.js

```diff
@@ -94,14 +170,98 @@ Page(
 				storage.setKey('token', req.params.value);
 
 				hmUI.showToast({
-					text: 'Token saved.', // + JSON.stringify(req.params),
+					text: 'Token saved ' + JSON.stringify(req.params),
 				});
 				storage.setKey('token', req.params.value);
 			}
 		},
+
+		onClickSleepButton() {
```

Comment:
in the future, when we add more permissions pages, I think it would be useful to separate this method (and other methods related to sleep here) to a different file/folder where all the sleep stuff is, so that this file doesn't become cluttered with a few 'unrelated' items